atomic_reactor/plugins/post_pulp_pull.py

```diff
@@ -84,4 +84,6 @@ def run(self):
                        image_id)
         self.workflow.builder.image_id = image_id
 
-        return image_id
+        # currently we only support v1+v2 schema1
```

Comment:
Consider the `adjust_for_registry_api_versions` method in `BuildRequest`: https://github.com/projectatomic/osbs-client/blob/master/osbs/build/build_request.py#L441
It embeds the following heuristic:

- If `pulp_push` ran, we expect `v1` content.
- If `pulp_sync` ran, we expect `v2` content (currently only schema 1).

Obviously, that is not sufficient since it cannot know whether schema 1 or schema 2 is being used for v2 content. IMO, this plugin should query the registry to figure out what exactly is there. The changes to `koji_import` should happen after that's done.